src/test/test_list.txt

```diff
@@ -0,0 +1,285 @@
+arith_uint256_tests/basics
+arith_uint256_tests/shifts
+arith_uint256_tests/unaryOperators
+arith_uint256_tests/bitwiseOperators
+arith_uint256_tests/comparison
+arith_uint256_tests/plusMinus
+arith_uint256_tests/multiply
+arith_uint256_tests/divide
+arith_uint256_tests/methods
+arith_uint256_tests/bignum_SetCompact
+arith_uint256_tests/getmaxcoverage
+addrman_tests/addrman_simple
+addrman_tests/addrman_ports
+addrman_tests/addrman_select
+addrman_tests/addrman_new_collisions
+addrman_tests/addrman_tried_collisions
+addrman_tests/addrman_find
+addrman_tests/addrman_create
+addrman_tests/addrman_delete
+addrman_tests/addrman_getaddr
+addrman_tests/caddrinfo_get_tried_bucket
+addrman_tests/caddrinfo_get_new_bucket
+addrman_tests/addrman_selecttriedcollision
+addrman_tests/addrman_noevict
+addrman_tests/addrman_evictionworks
+amount_tests/MoneyRangeTest
+amount_tests/GetFeeTest
+amount_tests/BinaryOperatorTest
+amount_tests/ToStringTest
+allocator_tests/arena_tests
+allocator_tests/lockedpool_tests_mock
+allocator_tests/lockedpool_tests_live
+base32_tests/base32_testvectors
+base58_tests/base58_EncodeBase58
+base58_tests/base58_DecodeBase58
+base64_tests/base64_testvectors
+abc/bech32_tests/bip173_testvectors_valid
```

Comment:
not sure why abc/ is appended to path. Get
```
Test setup error: no test cases matching filter
```
Removing abc/ lets tests run and pass.

Reply:
This also poses an interesting problem. If the test case isn't found, its listed as `FAILED TESTS`, which is somewhat confusing, because `bech32_tests/bip173_testvectors_valid` would pass, it's just the path listed is wrong. 

Maybe the initial reader of `FILE_TEST_LIST` can verify the path exists before handing it off to workers.